BSB_lan_defs.h

```diff
@@ -8363,13 +8435,18 @@ PROGMEM_LATE const cmd_t cmdtbl2[]={
 
 {0x073D082F,  CAT_IOTEST,           VT_VOLTAGE,       7854,  STR7854,  0,                    NULL,         FL_RONLY,     DEV_ALL}, // Spannungssignal H3
 {0x073D0809,  CAT_IOTEST,           VT_CLOSEDOPEN,    7855,  STR7855,  sizeof(ENUM_CLOSEDOPEN), ENUM_CLOSEDOPEN, FL_RONLY,     DEV_ALL}, // Kontaktzustand H3
-{0x053D174E,  CAT_IOTEST,           VT_UNKNOWN,       7858,  STR7858,  0,                    NULL,         FL_RONLY,     DEV_ALL}, // Eingangssignal H3
+
+// Elco Wärmepumpe RVS61.843E/560 - Signalart H1 [Kein ¦ Geschl' (ooo), Offen (---) ¦ Impulse ¦ Frequenz Hz ¦ Spannung V]
+// "---" == DC 80 0A 10 07 05 3D 17 4E 01 00 00 00 00 44 74 
+{0x053D174E,  CAT_IOTEST,           VT_CUSTOM_ENUM,   7858,  STR7858,  sizeof(ENUM7858),        ENUM7858,        FL_RONLY,     DEV_ALL}, // Eingangssignal H3
```

Comment:
Same as above.

Reply:
This input is not used/connected in my configuration.

Reply:
Ok, in that case, please set these parameters to VT_UNKNOWN because VT_CUSTOM_ENUM only returns a "correct" answer of "---" because the returned parameter is "disabled" (0x01). Once that is not the case, it will not return a correct value.